reactDashboard/src/components/RouteMapRules/RouteMapRules.css

```diff
@@ -4,6 +4,17 @@
   padding: 1rem;
 }
 
+.darkmode .headline-banner {
+  background-color: #133a53;
+}
+
+.darkmode .headline-banner h2 {
```

Comment:
Put this under the block it is overriding, and why does this block have a color override but .headline-banner h2 doesn't ?